### PR TITLE
Start workout

### PR DESCRIPTION
**Pull request by WennyXiong**

### CODE REVIEW

**Review by mmckenna34 — APPROVED** [review]
LGTM

**Review by samkbe — APPROVED** [review]
💯

File: client/src/components/StartWorkout/EndSessionModal.jsx [hunk 1]
```diff
@@ -1,5 +1,65 @@
...
+      url: `${process.env.URL}/updateWorkoutCompletion`,
+      headers: {
+        ContentType: 'application/json',
+        authorization: `Bearer ${Userfront.tokens.accessToken}`,
+      },
+      params: {
+        userId,
+        workoutId,
+        finishCount,
+        completeTime,
```
Comment (PatMan817) [inline client/src/components/StartWorkout/EndSessionModal.jsx]: Change to req.body

**Review by PatMan817 — CHANGES_REQUESTED** [review]
Change to req..body

**Review by PatMan817 — APPROVED** [review]

**Pull request merged**